asistente_ladm_col/logic/quality/logic_quality_rules.py

```diff
@@ -0,0 +1,494 @@
+from asistente_ladm_col.utils.utils import get_uuid_dict
+
+
+class LogicQualityRules:
+    def __init__(self, qgis_utils):
+        self.quality_rules_manager = QualityRuleManager()
+        self.qgis_utils = qgis_utils
+        self.logger = Logger()
+        self._ladm_queries = dict()
+
+    def get_ladm_queries(self, engine):
+        if engine not in self._ladm_queries:
+            self._ladm_queries[engine] = ConfigDBsSupported().get_db_factory(engine).get_ladm_queries(self.qgis_utils)
+
+        return self._ladm_queries[engine]
+
+    def check_parcel_right_relationship(self, db):
+        rule = self.quality_rules_manager.get_quality_rule(EnumQualityRule.Logic.PARCEL_RIGHT_RELATIONSHIP)
+        error_layer = None
+        error_layer_exist = False
+        group = self.qgis_utils.get_error_layers_group()
+        layers = group.findLayers()
+        for layer in layers:
+            if layer.name() == rule.error_table_name:
+                error_layer = layer.layer()
+                error_layer_exist = True
+                break
+
+        if error_layer is None:
+            error_layer = QgsVectorLayer("NoGeometry", rule.error_table_name, "memory")
+            pr = error_layer.dataProvider()
+            pr.addAttributes(rule.error_table_fields)
+            error_layer.updateFields()
+
+        new_features = list()
+        res, records = self.get_ladm_queries(db.engine).get_parcels_with_not_right(db)
+        if res:
+            for record in records:
+                new_feature = QgsVectorLayerUtils().createFeature(error_layer,
+                                                                  QgsGeometry(),
+                                                                  {0: record[db.names.T_ILI_TID_F],
+                                                                   1: self.quality_rules_manager.get_error_message(QUALITY_RULE_ERROR_CODE_E400102),
+                                                                   2: QUALITY_RULE_ERROR_CODE_E400102})
+                new_features.append(new_feature)
+
+        res, records = self.get_ladm_queries(db.engine).get_parcels_with_repeated_domain_right(db)
+        if res:
+            for record in records:
+                new_feature = QgsVectorLayerUtils().createFeature(error_layer,
+                                                                  QgsGeometry(),
+                                                                  {0: record[db.names.T_ILI_TID_F],
+                                                                   1: self.quality_rules_manager.get_error_message(QUALITY_RULE_ERROR_CODE_E400101),
+                                                                   2: QUALITY_RULE_ERROR_CODE_E400101})
+                new_features.append(new_feature)
+
+        error_layer.dataProvider().addFeatures(new_features)
+        return self.return_message(db, new_features, rule.error_table_name, error_layer, error_layer_exist)
```

Comment:
From the comment above, evaluate if the last parameter "error_layer_exist" is necessary in `self.return_message()`.

Reply:
Thank you, this was resolved in e9e311a 
Another parameter was removed that was not necessary.